Modules/System/File Access/src/Connector/FileConnector.Interface.al

```diff
@@ -0,0 +1,144 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// An file connector interface used to creating file accounts and handle external files.
+/// </summary>
+interface "File Connector"
+{
+    /// <summary>
+    /// Gets a List of Files stored on the provided account.
+    /// </summary>
+    /// <param name="AccountId">The file account ID which is used to get the file.</param>
+    /// <param name="Path">The file path to list.</param>
+    /// <param name="Files">A list with all files stored in the path.</param>
+    procedure ListFiles(AccountId: Guid; Path: Text; var FileAccountContent: Record "File Account Content" temporary);
+
+    /// <summary>
+    /// Gets a file from the provided account.
+    /// </summary>
+    /// <param name="AccountId">The file account ID which is used to get the file.</param>
+    /// <param name="Path">The file path inside the file account.</param>
+    /// <param name="Stream">The Stream were the file is read to.</param>
+    procedure GetFile(AccountId: Guid; Path: Text; Stream: InStream);
```

Comment:
InStream are always passed by reference, but I'd prefer explicit delcaring it as var.

Reply:
I like to keep code as short as possible. I checked the System app and I find no fix rule. TempBlob Impl. uses var. The Image and Email Attachment not :(